modules/server_new/src/main/scala/observe/server/ObserveEngineImpl.scala

```diff
@@ -1344,15 +1349,15 @@ private class ObserveEngineImpl[F[_]: Async: Logger](
     i: (EventResult, EngineState[F])
   ): F[(EventResult, EngineState[F])] =
     (i match {
-      case (SystemUpdate(SystemEvent.Failed(id, _, e), _), _) =>
-        Logger[F].error(s"Error executing $id due to $e") <*
+      case (SystemUpdate(SystemEvent.Failed(obsId, _, e), _), _) =>
+        Logger[F].error(s"Error executing $obsId due to $e") <*
           systems.odb
-            .stepAbort(id)
+            .stepAbort(obsId)
```

Comment:
Does this notify the client? I think the client wants the obs reference to be displayed rather than the id

Reply:
This is the ODB notification. I agree with you, I'll look into it in another PR.